docs/src/content/docs/samples/chain-of-debate.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: Chain of Debate
+description: Multi-agent debate system where LLM models engage in structured, iterative debates
+sidebar:
+    order: 15
+---
+
+import { Code } from "@astrojs/starlight/components"
+import source from "../../../../../samples/sample/genaisrc/chain-of-debate.genai.mts?raw"
+import sourceSimple from "../../../../../samples/sample/genaisrc/chain-of-debate-simple.genai.mts?raw"
+
+This sample implements a **chain of debate** pattern where multiple LLM models engage in structured, iterative debate on a given topic. The implementation allows multiple AI agents to present initial positions, then respond to each other's arguments over several rounds, culminating in a synthesized conclusion.
```

Comment:
Change the tone of the article so that it reads more like story telling /blogging. Less like a bullet point list.

Don't be too excited but point out the interresting bits.